gatsby/src/components/ConferenceSpeaker.js

```diff
@@ -4,8 +4,6 @@ import Img from 'gatsby-image';
 import styled from 'styled-components';
 
 const OnlineSeriesTalk = styled.div`
```

Comment:
It looks like this styled div is not being used anymore. the class .talk-time isn't assigned to anything in the code base. Should we just go ahead and remove this?

Reply:
If we did remove this, we could then just need to remove the <OnlineSeriesTalk> elements down below.